Makefile

```diff
@@ -0,0 +1,6 @@
+init:
+	pip install pipenv
+	pipenv install
+
+test:
+	pipenv run pytest tests/.
```

Comment:
I have no idea about make files, but I would have expected this to be a step in the travis config

Reply:
Instead of running it from .travis.yml it will be handled by make file